packages/time-picker/src/TimePicker/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import { useTreeState, TreeProps } from "@react-stately/tree"
+import { AriaMenuOptions, useMenu } from "@react-aria/menu"
+import { MenuItem } from "../MenuItem"
+import { TIME_OPTION } from "../../utils"
+import styles from "./Menu.module.scss"
+
+export const Menu = (props: AriaMenuOptions<TIME_OPTION> & TreeProps<any>) => {
+  const state = useTreeState(props)
+  const ref = React.useRef(null)
+  const { menuProps } = useMenu(props, state, ref)
+
+  return (
+    <ul {...menuProps} ref={ref} className={styles.menuContainer}>
+      {/* @ts-ignore */}
```

Comment:
Whatever this is, it's probably not something we'd want to ignore? If there's a good reason though, make sure to put a comment to explain it :)

Reply:
Ooh yeah sorry there's going to be a few of these scattered around. Haven't gotten around to cleaning it up yet